### PR TITLE
Remove the "This module" suffix and reword some entries from the stdlib overview

### PR DESCRIPTION
**Pull request by ghost**
I feel like the stdlib overview is too wordy and uses "This module x" despite it being kind of obvious that the description is about the module itself.

This is a draft for now, since I don't know if this is a welcome change or not, and I also changed a few of the entries beyond just removing the prefix.

Comments/edits to the PR welcome

### CODE REVIEW

File: doc/lib.md [hunk 1]
```diff
@@ -254,52 +253,50 @@ Generic Operating System Services
...
 
 * [syncio](syncio.html)
-  This module implements various synchronized I/O operations.
+  Various synchronized I/O operations.
 
 * [terminal](terminal.html)
-  This module contains a few procedures to control the *terminal*
-  (also called *console*). The implementation simply uses ANSI escape
-  sequences and does not depend on any other module.
+  A module to control the terminal output (also called *console*).
```
Comment (ghost) [inline doc/lib.md:281]: I feel like the old description was too wordy and belittled the module - it is not "a few procedures" anymore :)

File: doc/lib.md [hunk 2]
```diff
@@ -324,75 +321,69 @@ Internet Protocols and Support
...
 * [net](net.html)
-  This module implements a high-level sockets API. It replaces the
-  `sockets` module.
+  A high-level sockets API.
 
 * [selectors](selectors.html)
-  This module implements a selector API with backends specific to each OS.
-  Currently, epoll on Linux and select on other operating systems.
+  A selector API with backends specific to each OS.
+  Supported OS primitives: `epoll`, `kqueue`, `poll`, and `select` on Windows.
```
Comment (ghost) [inline doc/lib.md:365]: Not sure if the primitives part is even needed, but I decided to just update it to reflect the newest changes instead of removing altogether

File: doc/lib.md [hunk 3]
```diff
@@ -637,7 +627,7 @@ Generic Operating System Services
 ---------------------------------
 
 * [rdstdin](rdstdin.html)
-  This module contains code for reading from stdin.
+  Code for reading user input from stdin.
```
Comment (ghost) [inline doc/lib.md:630]: i think the old description was too generic since you could already read from stdin with `stdin.readLine`, so I at least added this, maybe it's worth explaining it even further

**Comment by Araq** [comment]
Much appreciated, undraft it already!
